### DISCUSSION TITLE
Meetings resisting (general location id)

### DISCUSSION THREAD

**Discussion by k8fish3r** [code4recovery/12-step-meeting-list#1576] [General]
Hullo

have a pattern I'm seeing in the fellowship I support - I'm web dev like person. 

Wondered if anyone else has seen/managed support any of these functions? Within 12 step meeting listing component? 

**Symptoms**
1). Virtual meetings resistent to "approximate location" 
- changing their name to (amorphous descriptions)
- meetings teams "prefer to welcome members from any region" (most virtual meetings 65% of all meetings sit as virtual with no location) 

2). Venue temporarily closed
- meetings still using  "**venue  temporarily closed**" which includes a physical region info
- I suspect 12sml "approximate location" is our way out - as its the most accurate? 

3).Masking meeting join link
We used to use generic zoom link to work around this. But that's not working now.
E.g. https://zoom.us/, https://zoom.com/
Is there a way to list virtual meetings without a join link?

Any ESH, and technical wizardry very much more than welcome. To broaden discussions from single perspectives of one fellowship and region.

**Comment by ginslo** [comment]
Hi @k8fish3r 

1. The approximate location is mainly useful to establish where the meeting originates. Those joining the meeting are welcome to join from anywhere of course as you mentioned.

2. The "venue temporarily closed" type is there when the meeting listed has a specific address. To remove that type, enter a general address. There is a warning that comes up in the Wordpress editor that says this when a specific address is entered for online-only meetings:
Example:
Location: Online-Philadelphia
Address: Philadelphia, PA, USA

3. When a meeting is hesitant to allow public access, we recommend continuing to keep the ID in the link while withholding the password. This can be done while providing a contact method (email/phone etc) to request the password.
